tests/by-util/test_ls.rs

```diff
@@ -567,6 +567,21 @@ fn test_ls_long_ctime() {
     assert!(result.stdout.contains("???"));
 }
 
+#[test]
+fn test_ls_order_birthtime() {
+    let scene = TestScenario::new(util_name!());
+    let at = &scene.fixtures;
+
+    at.touch("test-birthtime-1");
+    at.touch("test-birthtime-2");
```

Comment:
Also, the tests seem to fail, probably because there is almost no time between the creation of these files, so maybe sleeping for 500 milliseconds would help?

Reply:
> there is almost no time between the creation of these files
This was my guess at the problem as well.